test/winter_onboarding_2021/fleet_management/models/cab_test.clj

```diff
@@ -0,0 +1,28 @@
+(ns winter-onboarding-2021.fleet-management.models.cab-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [winter-onboarding-2021.fleet-management-service.models.cab :as cab]
+            [winter-onboarding-2021.fleet-management.fixtures :as fixtures])
+  (:import [org.postgresql.util PSQLException]))
+
+(use-fixtures :once fixtures/config fixtures/db-connection)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest create-cab
+  (testing "Should add a cab"
+    ;; NOTE: the cab/create input needs to be auto kebab'ised, need to use honeysql for this
```

Comment:
Isn't this already taken care off?

Reply:
Currently, only the output from database is kebab'ised.
Will fix this.